client/ayon_traypublisher/api/plugin.py

```diff
@@ -316,11 +316,12 @@ def from_settings(cls, item_data):
         identifier = item_data["identifier"]
         product_type = item_data["product_type"]
         if not identifier:
-            identifier = "settings_{}".format(product_type)
+            identifier = f"settings_{product_type}"
         return type(
-            "{}{}".format(cls.__name__, identifier),
+            f"{cls.__name__}{identifier}",
             (cls, ),
             {
+                "product_base_type": product_type,
```

Comment:
So the customization of this through settings for SimpleCreators would be in a follow up PR later?

Reply:
Yes, when product base type is a must have then we ~~should~~ have to add it to settings.